components/NumberInput.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <v-text-field
+    :value="value"
+    @input="$emit('input', $event)"
+    type="number"
+    :min="min || null"
+    :max="max || null"
```

Comment:
The default value for these is already null.  Is there a reason why we need the `|| null` here as well?